automated-comments/elixir/general/parsing_error.md

```diff
@@ -0,0 +1,14 @@
+<!--
+requires 3 params:
+  line: line of the error
+  error: the error encountered
+  token: if a specific token is mentions by the parser
```

Comment:
```suggestion
  token: if a specific token is mentioned by the parser
```

Reply:
token now concatenated with error in analyzer.